bundles/org.openhab.binding.insteon/pom.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.openhab.addons.bundles</groupId>
+    <artifactId>org.openhab.addons.reactor.bundles</artifactId>
+    <version>2.5.2-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>org.openhab.binding.insteon</artifactId>
+
+  <name>openHAB Add-ons :: Bundles :: Insteon Binding</name>
+
+  <dependencies>
+    <dependency>
+      <groupId>org.apache.httpcomponents</groupId>
+      <artifactId>httpclient</artifactId>
```

Comment:
Any reason not to use the openHAB httpclient component? Or was this in the oh1 version of the binding? It would be great if you could use the openHAB httpclient instead.

Reply:
The oh1 binding used this version of httpclient. Since I only have a plm which uses a serial port, I didn't want to change the library to something else.

Reply:
The ported code currently calls a method that was removed in later versions of httpclient. Instead of refactoring the code, a decision was made to stick with httpclient 4.1.3. This is needed to communicate with a hub (which I don't have). 

See https://github.com/openhab/openhab1-addons/issues/3922

Reply:
The code was refactored and this has been removed.